scale/ingest/container.py

```diff
@@ -0,0 +1,9 @@
+"""Defines the fields for handling file systems in the local container volumes for ingest and Strike jobs"""
+from __future__ import unicode_literals
+
+import os
+
+from storage.container import SCALE_ROOT_PATH
+
+
+SCALE_INGEST_MOUNT_PATH = os.path.join(SCALE_ROOT_PATH, 'ingest_mount')
```

Comment:
It seems like these types of declared paths should go inside an app-specific settings.py file.